Code/Mantid/Framework/DataHandling/inc/MantidDataHandling/LoadMask.h

```diff
@@ -115,16 +115,16 @@ class DLLExport LoadMask : public API::Algorithm {
                                std::vector<int> &rangeendvec);
 
   /// Mask Workspace
-  DataObjects::MaskWorkspace_sptr mMaskWS;
+  DataObjects::MaskWorkspace_sptr m_MaskWS;
```

Comment:
I suspect when you renamed this, you didn't mean to capitalise the `M` :)

Same goes for `m_DefaultToUse`.